### DISCUSSION TITLE
DefectDojo Jira Integration

### DISCUSSION THREAD

**Discussion by rbalajibre** [DefectDojo/django-DefectDojo#9507] [General]
Integrating with Jira local server: Jira status is not updating in DefectDojo.
![Jira1](https://github.com/DefectDojo/django-DefectDojo/assets/52751550/12778a98-b0e1-4036-bc26-2614bb51543b)
![jira2](https://github.com/DefectDojo/django-DefectDojo/assets/52751550/30f7dc0b-b263-4a4d-b509-67830b38c6b6)

**Comment by mtesauro** [comment]
Bi-directional support requires webhook integration from Jira to DefectDojo
https://github.com/DefectDojo/django-DefectDojo/discussions/9507
